Market-Trading-System/src/main/java/DomainLayer/Market/User/ShoppingCart.java

```diff
@@ -0,0 +1,31 @@
+package DomainLayer.Market.User;
+
+public class ShoppingCart {
+    private InMemory<ShoppingBasket> baskets;
+
+    public ShoppingCart(){
+        baskets = new InMemory<ShoppingBasket>();
+    }
+
+    public String viewShoppingCart(){
+        String res = "Your shopping cart contains the following baskets:\n";
```

Comment:
The message shouldn't be hard coded put only the items as string

Reply:
fixed